### PR TITLE
feat: HIP-904: Implement TokenClaimAirdrop transaction

### PR DESCRIPTION
**Pull request by vtronkov**
**Description**:
Add TokenClaimAirdrop basic implementation

**Notes for reviewer**:
- Add configuration
- Add throttles.json values
- Add TokenClaimAirdrop empty shell - we always throw a unsupported exception until we merge the actual implementation of the feature flag

### CODE REVIEW

**Review by MrValioBg — APPROVED** [review]
LGTM! Thank you very much!

Just a nit - consider re-checking function params and making them final / @NonNull (We could do it in a following PR with the implementation.)

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM!

File: hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json [hunk 1]
```diff
@@ -69,7 +69,8 @@
 			"TokenFreezeAccount",
 			"TokenUnfreezeAccount",
 			"TokenGrantKycToAccount",
-			"TokenRevokeKycFromAccount"
+			"TokenRevokeKycFromAccount", 
+      "TokenClaimAirdrop"
```
Comment (stoyanov-st) [inline hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json:73]: Could you please adjust the indentation here?

File: hedera-node/hedera-file-service-impl/src/main/resources/genesis/throttles-dev.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
 			"CryptoCreate",
 			"CryptoTransfer",
             "TokenAirdrop",
+				"TokenClaimAirdrop",
```
Comment (stoyanov-st) [inline hedera-node/hedera-file-service-impl/src/main/resources/genesis/throttles-dev.json:13]: Also adjust here.

**Review by stoyanov-st — APPROVED** [review]
LGTM 🚀 , just a small nit for the json indentations

File: hedera-node/hedera-file-service-impl/src/main/resources/genesis/throttles-dev.json (hunk 2 above)
Comment (vtronkov) [reply hedera-node/hedera-file-service-impl/src/main/resources/genesis/throttles-dev.json]: Fixed 👍

File: hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json (hunk 1 above)
Comment (vtronkov) [reply hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json:73]: Not sure what's going on but in Github it looks weird but when I open it in IntelliJ it looks fine 🤷

Comment (stoyanov-st) [reply hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json:73]: Is uses spaces instead of tabs

Comment (vtronkov) [reply hedera-node/test-clients/src/main/resource/testSystemFiles/throttles-dev.json]: Now it should be ok 😄

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM! Thanks @vtronkov

**Review by nathanklick — APPROVED** [review]
Approved as to form only. No assurances have been made as the substance of these changes.

**Pull request merged**